### PR TITLE
NO-JIRA [Do not merge] Fix build

### PR DESCRIPTION
**Pull request by 7PH**

### CODE REVIEW

File: src/extensions/sonarcloud/vss-extension.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "sonarcloud",
   "name": "SonarCloud",
-  "version": "1.46.0",
+  "version": "5.50.11",
```
Comment (lucas-paulger-sonarsource) [inline src/extensions/sonarcloud/vss-extension.json:5]: afaik, we can bump the task version without needing to bump extension version and it should still bust the task cache properly.

I wanted to suggest that we could implement bumping the patch version per run. and when we release we bump minor and reset the patch back to 0

Comment (lucas-paulger-sonarsource) [reply src/extensions/sonarcloud/vss-extension.json:5]: my guess is that the assumption was by deleting and re-installing the extension for ITs , it was clearing the cache.

Comment (lucas-paulger-sonarsource) [reply src/extensions/sonarcloud/vss-extension.json]: you would think the extension cache would supersede the task cache but it seems that's not the case

**Comment by 7PH** [comment]
Issue fixed itself. Closing this PR unless the issue happens again, but I'd monitor QA in the next runs to be sure it doesn't happen again.

**Pull request closed without merging**